### DISCUSSION TITLE
Add "Schema Review" tool

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5862] [Feature Requests]
It would be very helpful to see what fields exist on the forms so you can make an informed decision on what should be hidden and what should not be.

## In Specify 6
<img width="492" alt="image" src="https://github.com/specify/specify7/assets/37256050/84fe2d49-cf99-4a0f-8f5e-a0add6a2a3f5">
<img width="1002" alt="image" src="https://github.com/specify/specify7/assets/37256050/849dde62-97a7-471d-b292-2cac111b5b60">

**Comment by maxpatiiuk** [comment]
how is "Is on Form" defined?
is on any form?
is on user defined form?
is on user defined + default form?
is on any form in the scope of current user?

also, this tool would be much less useful once Form Editor V2 is implemented - there it would be easy to warn you if you are using a hidden field, or to show you a list of fields you haven't yet added to the form

thus, my suggestion would be to no re-implement this tool in sp7 in the present form but to integrate it's core features into form editor v2
